src/stirling/source_connectors/socket_tracer/socket_trace_connector.h

```diff
@@ -93,6 +93,7 @@ class SocketTraceConnector : public SourceConnector, public bpf_tools::BCCWrappe
   static constexpr auto kTables =
       MakeArray(kConnStatsTable, kHTTPTable, kMySQLTable, kCQLTable, kPGSQLTable, kDNSTable,
                 kRedisTable, kNATSTable, kKafkaTable, kMuxTable, kAMQPTable);
+  // -- TODO -- remove -- static constexpr auto kDataTableSchema = kTables;
```

Comment:
Can this be removed?

Reply:
Yes, removed. (that was not supposed to be there!).